.mergify.yml

```diff
@@ -11,9 +11,9 @@ pull_request_rules:
         message: |
           Thanks for submitting this pull request.
           Bors-ng will now build test images. When it succeeds, we will continue to review and test your PR.
-          
```

Comment:
Aren't these indentations required by the yaml spec? Afaik | continues until indentation changes.

Reply:
Yes. These indentations are required to continue the multiline string.